lib/control-plane/openstackcontrolplane.yaml

```diff
@@ -35,7 +37,7 @@ spec:
           - storage
         replicas: 0
       cinderScheduler:
-        replicas: 1
+        replicas: 3
```

Comment:
I think cinder-scheduler should be 1, but @Akrog to confirm

Reply:
I confirm, that's what we are recommending for production because it's going to solve a bunch of issues for deployments.
TBH for CI shouldn't matter, but I think it has value to test things following our recommendations.

Reply:
reading the document linked i agree keeping it at 1 for now is best.
with that said i think there is definitely an upstream conversation to be had about finally making cinder aware of placement and using it to track allocation against backends so that you can move form eveutally consists to a atomic allocations model to allow horizontal scaling

that will take time if the cinder community want to move in that direction so
for 18/antelope replica 1 make sense to me. this is not something we would expect to change in the lifetime of 18.

Reply:
set cinder-scheduler to